src/app/docs/kagent/examples/documentation/page.mdx

```diff
@@ -199,6 +251,39 @@ Click on `Add Tools`.
 
 Select the `query-documentation` tool.
 
+### Option 2: Using YAML CRDs
+
+Create an `agent.yaml` file:
+
+```yaml
+# Kagent Agent Configuration
+# This defines an AI agent that can answer questions about MCP documentation
+apiVersion: kagent.dev/v1alpha1
+kind: Agent
```

Comment:
The Agent API has also changed in prep for v0.6. This would now need a `type: Declarative` at the level of description/modelConfig, and then indent the settings under `declarative` field. And the way you refer to mcpServers is also updated.

Reply:
I see, let me then install the pre-release and make another update.